XamlControlsGallery/ControlPages/RevealFocusPage.xaml

```diff
@@ -92,10 +92,9 @@
                 </StackPanel>
```

Comment:
This page is missing UIA names that I'm sure I added. Probably just haven't been merged in from Master.

Reply:
I was looking at the wrong branch. The names are where they should be.